src/plugins/home/public/application/components/tutorial_directory.js

```diff
@@ -59,6 +62,9 @@ const homeTitle = i18n.translate('home.breadcrumbs.homeTitle', { defaultMessage:
 const addDataTitle = i18n.translate('home.breadcrumbs.addDataTitle', {
   defaultMessage: 'Add data',
 });
+const localCluster = i18n.translate('home.dataSource.localCluster', {
+  defaultMessage: 'Local Cluster',
+});
```

Comment:
np: do we want the data source picker in dev tool console to have the same local cluster option displayed? I am wondering what could be generic solution for everything that we want local cluster rendered.

Reply:
There are 2 ways I think we could do about this:

- We can either insert it in the public datasourceManagement GetDatasources util, Or
- Add it in the data source saved object wrapper level 

However, with Meta storage feature, this will on longer be needed.